common/unified-chart/templates/high-priority/analysis-template.yaml

```diff
@@ -0,0 +1,28 @@
+{{- $top := . -}}
+{{- $values := .Values -}}
+{{- if .Values.argoRollouts.progressiveDelivery.enabled }}
+---
+apiVersion: argoproj.io/v1alpha1
+kind: AnalysisTemplate
+metadata:
+  name: black-box
+spec:
+  args:
+    - name: service-name
+      value: {{ include "hp.name" . }}
+  metrics:
+    - name: succesful-black-box-test
+      successCondition: result == true
+      provider:
+        web:
+          method: POST # valid values are GET|POST|PUT, defaults to GET
+          url: {{ .Values.argoRollouts.progressiveDelivery.blackBox.url | default "http://nexus.default.svc.cluster.local:8080/blackbox" }}
```

Comment:
why do we need 8080 on a service?

Reply:
@guy-frontegg We have two open ports in Nexus, one of the app and another for the metrics

Reply:
would be a good idea to change svc of nexus to listen on 80
so we wont have to specify ports everywhere
also unity
all other services we have use port 80

Reply:
why not map it in the service